etc/avocado/avocado.conf

```diff
@@ -19,6 +19,8 @@ enabled = True
 installed_packages = False
 # Whether to run certain commands in bg to give extra job debug information
 profiler = False
+# Force LANG for sysinfo collection
+lang = C
```

Comment:
And a final comment for completeness and to make it extra clear: LC_ALL overrides all LC_* variables, while LANG is a fallback in case one is not set (LANG is an ancient variable and works as a fallback for compatibility purposes).